apps/admin_api/test/admin_api/v1/controllers/admin_auth/export_controller_test.exs

```diff
@@ -67,6 +67,19 @@ defmodule AdminAPI.V1.ExportControllerTest do
       assert response["success"]
       assert Enum.empty?(exports)
     end
+
+    test "does not return exports made with a different adapter" do
+      user = get_test_admin()
+      insert(:export, adapter: "local", user_uuid: user.uuid)
+      insert(:export, adapter: "local", user_uuid: user.uuid)
+      insert(:export, adapter: "gcs", user_uuid: user.uuid)
+
+      response = admin_user_request("/export.all", %{})
+      exports = response["data"]["data"]
+
+      assert response["success"]
+      assert length(exports) == 2
```

Comment:
Can you assert the two records for `adapter: "local"` too? Just in case the results are a side effect from other tests.